api/conf/conf.go

```diff
@@ -78,12 +84,38 @@ func setConf() {
 		if dagLibPath != "" {
 			DagLibPath = dagLibPath
 		}
+
 		//etcd
 		eTCDEndpoints := configuration.Get("conf.etcd.endpoints").String()
 		if eTCDEndpoints != "" {
 			ETCDEndpoints = eTCDEndpoints
 		}
 
+		//log
+		logLevel := configuration.Get("conf.log.error_log.level").String()
```

Comment:
can we use JSONSchema to set the default value?

Reply:
if set the default value by config, I think don't need JSONSchema, just set default value in conf file.